docs/UserGuide.md

```diff
@@ -26,7 +26,8 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
 
    * **`list`** : Lists all contacts.
 
-   * **`add`**`n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01` : Adds a contact named `John Doe` to the Address Book.
+   * **`add`** **'c/P n/John p/98765432 e/john@example.com g/M a/Bishan street, block
+     123, #01-01 t/Asthma d/2022-12-12 1350`** : Adds a patient named John to Healthcare Xpress.
```

Comment:
I think that on line 29, the wrong ` is used which caused it to not code block correctly.

Maybe we should add a code block to John's name?

Reply:
thanks for catching this mistake!